bin/common.sh

```diff
@@ -106,7 +106,13 @@ function getZeppelinVersion(){
     exit 0
 }
 
-# Text encoding for 
+function downloadSparkBinary() {
+  if [[ -z "${SPARK_HOME}" ]]; then
```

Comment:
@Leemoonsoo Thank you for pointing the possible issues! Didn't know that parts actually. Checking the existence of `${ZEPPELIN_HOME}/interpreter/spark` more makes sense than `SPARK_HOME`. And getting user's answer as well. Will update that two parts :)